tests/bud/copy-globs/Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM php:7.2
-COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
+FROM busybox
+COPY --from=alpine /we/never/get/here /nor/here/either
```

Comment:
If I'm reading "bud with .dockerignore #3" correctly, this was expected to succeed.

Reply:
Well, it does, but only because `Dockerfile` is ignored: as best I can tell, the test seems to use `Containerfile` instead. I don't know why there's a `Dockerfile` in that directory, maybe as a test that `Containerfile` will override it?

Reply:
Argh, I was looking at the `copy-from` directory.  Then I think this is right.